lib/elixir_internal_certificate/scraper/schemas/search_result.ex

```diff
@@ -9,13 +13,24 @@ defmodule ElixirInternalCertificate.Scraper.Schemas.SearchResult do
     field :html_response, :string
 
     belongs_to :user_search,
-               ElixirInternalCertificate.Scraper.Schemas.UserSearch,
+               UserSearch,
                foreign_key: :user_search_id
 
     has_many :url_results,
-             ElixirInternalCertificate.Scraper.Schemas.UrlResult,
+             UrlResult,
              foreign_key: :search_result_id
 
     timestamps()
   end
+
+  def search_result_changeset(search_result \\ %__MODULE__{}, attrs),
```

Comment:
normally we name the changeset as `create_changeset` or `update_changeset` if the changeset can shared between create and update, we will just name it as `changeset` for that reason, should we change this method Tue, what do you think?

Reply:
kept in mind 🙏

Reply:
Fixed in dbda9b9